ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/openshift-image-based-upgrade-e2e-conf-commands.sh

```diff
@@ -15,13 +15,27 @@ remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
 instance_ip=$(cat ${SHARED_DIR}/public_address)
 host=$(cat ${SHARED_DIR}/ssh_user)
 ssh_host_ip="$host@$instance_ip"
-TARGET_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
-target_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${TARGET_VM_NAME}/auth/kubeconfig
+
+TEST_VM_NAME="unknown"
+case $TEST_CLUSTER in
```

Comment:
This can also be adapted like https://github.com/openshift/release/pull/55000/files#diff-b3a2fd7afbf13799bb36dba46b1e57f492ee0aff78b4d390e78c049c4928158fR20.